app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -443,9 +443,11 @@ fun PocketStoriesCategories(
             verticalItemsSpacing = 16.dp,
         ) {
             categories.filter { it.name != POCKET_STORIES_DEFAULT_CATEGORY_NAME }.forEach { category ->
+                var selectedState by remember { mutableStateOf(selections.map { it.name }.contains(category.name)) }
```

Comment:
```suggestion
                var isSelected by remember { mutableStateOf(selections.map { it.name }.contains(category.name)) }
```

Reply:
Thinking about this some more - is this mutable state necessary here since we remember the state inside of SelectableChip as well?

Reply:
we don't strictly need this to be mutable if we're not referring to the changed state anywhere, I'll take a further look